### PR TITLE
docs(play): explain Play's hosting

### PR DESCRIPTION
**Pull request by HanselIdes**
## Description

Explain how Play is hosted

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -73,4 +73,5 @@ For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true`
...
 
 - Play runs on community-built projects, as described in the [Zeebe-Play repository](https://github.com/camunda-community-hub/zeebe-play).
-- Play is run on completely isolated infrastructure from the core Camunda deployment. It can only receive information from Web Modeler, and can only communicate externally using Connectors and the user-defined secrets in the secret store or BPMN diagram. Play is not authenticated, so anyone with the URL can access it. To mitigate this risk, there is a 34-character randomly generated UUID in the URL. Each session lasts approximately 20 minutes and the Play environment and its data are automatically deleted at the end of each session. You can reset the session timer by re-opening Play.
+- Play is run on completely isolated Camunda-hosted infrastructure from the core SaaS or Self-Managed Camunda deployment. It can only receive information from Web Modeler, and can only communicate externally using Connectors and the user-defined secrets in the secret store or BPMN diagram. Play is not authenticated, so anyone with the URL can access it. You should not submit personal or confidential information to Play.
```
Comment (akeller) [inline docs/components/modeler/web-modeler/play-your-process.md:76]: I recommend bringing more attention to the following two sentences:

"Play is not authenticated, so anyone with the URL can access it. You should not submit personal or confidential information to Play."

You could do this with an admonition.

**Comment by akeller** [comment]
I have a few initial questions here: 

Should we link to Legal resources like the [terms](https://legal.camunda.com/licensing-and-other-legal-terms) or [privacy statement](https://legal.camunda.com/privacy-and-data-protection)?

Should we direct users to the Contact page, forum, or support if they have more questions or concerns?

File: docs/components/modeler/web-modeler/play-your-process.md (hunk 1 above)
Comment (HanselIdes) [reply docs/components/modeler/web-modeler/play-your-process.md:76]: Sure, that works for me. Could you please help me out by making this change?

**Comment by HanselIdes** [comment]
> I have a few initial questions here:
> 
> Should we link to Legal resources like the [terms](https://legal.camunda.com/licensing-and-other-legal-terms) or [privacy statement](https://legal.camunda.com/privacy-and-data-protection)?
> 
> Should we direct users to the Contact page, forum, or support if they have more questions or concerns?

That's a good idea. https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms is the most relevant terms to most users. 

I don't expect many questions about this, so I don't think adding a link is necessary. If you feel we _do_ want a link, then I'd prefer people to join the C8 Slack and discuss Play in [this channel](https://camunda-platform.slack.com/archives/C04BY4B5SGZ). However, this would require the user to create a C8 slack account first.

**Review by akeller — APPROVED** [review]

**Pull request merged**